tests/snapshot/snapshots/snapshot__error_stderr_array_at_empty_array.ncl.snap

```diff
@@ -0,0 +1,29 @@
+---
+source: tests/snapshot/main.rs
+expression: err
+---
+error: contract broken by the caller of `at`: invalid array indexing
+    ┌─ <stdlib/std.ncl>:162:9
+    │
+162 │       | std.contract.unstable.IndexedArrayFun
+    │         ------------------------------------- expected type
+    │
```

Comment:
This block is given a lot of importance visually in my opinion, but the information I get from it is kind of not terribly helpful because I don't know what `std.contract.unstable.IndexedArrayFun` is or means. The blame message shown ("Can't index into an empty array") is very helpful but my eyes don't jump to it immediately. Maybe it looks different with colors.
Either way, this isn't a blocker, this is a good improvement already.

Reply:
One solution would be to set the "Can't index into an empty array" at the top with the main label ("invalid array index"), although in general I try to keep the main label more generic and the notes more specific. But I agree with you.

I don't have a great solution right now on the top of my head: underlining the failing contract is very useful in most cases, and it's a generic process (it's not controlled by the contract implementation). I wanted to open an issue to have a dedicated syntax for dependent function contracts; maybe in that case, we wouldn't have those kind of "opaque" contracts anymore, and the underlining would be more useful. You can imagine something like:

```
Number ->dep contract.IndexedArray -> Dyn
```

with `contract.IndexedArray` being underlined.